src/lib/components/ui/Search.svelte

```
@@ -0,0 +1,206 @@
+										class="flex items-center gap-2 divide-x divide-neutral-40 group-hover:divide-neutral-80 text-xs mt-1.5 text-neutral-40 group-hover:text-neutral-80 font-mono"
+									>
+										<p>20 Jan 2024</p>
+										<p class="pl-2">5 min read</p>
+									</div>
+								</li>
+							</ul>
+						</div> -->
+
+						<ul class="mt-2 text-left">
+							{#each $results as article}
+								{@const highlight = article?._highlightResult?.content_excerpt as any}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)